messages_appendix.cddl

```diff
@@ -332,7 +338,7 @@ remote-playback-state-event = {
 remote-playback-id = uint
 
 remote-playback-controls = {
-  ? 1: url ; source
+  ? 1: text ; source-url
```

Comment:
Are these changes related?  I think I prefer the "url" semantic type for these fields.

Reply:
"url" is a type that we used to have be removed at some point.  Whenever we removed it, we made a mistake here in not changing this.

If we want to re-add the url type, we can do that in a separate RP.